roles/deploy/tasks/main.yml

```diff
@@ -78,6 +78,13 @@
   when: s3_backups_bucket != ""
   tags: symlink
 
+- name: install bundler
+  # This needs to be run inside a bash shell to initialise rbenv
+  # See http://stackoverflow.com/questions/22115936/install-bundler-gem-using-ansible
+  command: bash -lc "RBENV_VERSION='{{ ruby_version }}' gem install bundler --no-ri --no-rdoc -v \"$(grep -m 1 -A 1 -x -F 'BUNDLED WITH' '{{ build_path }}/Gemfile.lock' | tail -n 1  | tr -d '[:space:]')\""
```

Comment:
Is it possible to store the parsing of the version in a var? And perhaps that's better suited in a role. It'd increase readability a great deal.

Reply:
I thought about that. This was just a quick fix to help Shawn, but you are right that we should solve it properly.

I revived a bundler install script that we had in the OFN repo a while ago. That script is much more useful and can be used by devs as well. It also makes sure that it upgrades or downgrades correctly. Have a look at https://github.com/openfoodfoundation/openfoodnetwork/pull/3302 and my new commit here.